pkg/eventbus/event_bus.go

```diff
@@ -35,19 +46,39 @@ func (eb *eventbus) Start(c *context.Context) {
 	// no need to call TopicInit now, we have fixed topic
 	eb.context = c
 
-	mqttURL := config.CONFIG.GetConfigurationByKey("mqtt.server")
+	mqttURL := config.CONFIG.GetConfigurationByKey("mqtt.internal-mqtt")
 	nodeID := config.CONFIG.GetConfigurationByKey("edgehub.controller.node-id")
+
+	thirdpartyMqttUrl := config.CONFIG.GetConfigurationByKey("mqtt.thirdparty-mqtt")
 	if mqttURL == nil || nodeID == nil {
```

Comment:
@kexun , if mode is 2, i.e external mqtt mode, we may not give mqtt.internal-mqtt in config. Is this panic required in that case ??